docs/maintenance/release-management.md

```diff
@@ -146,3 +146,16 @@ only need to replace ``master`` with ``maint-x.y`` branches.
     Be aware that when the independent modules are released, they will be
     picked up immediately by new InvenioRDM installations. This should not be
     an issues since the releases MUST be backward compatible.
+
+## Bleeding Edge Development Workflow
```

Comment:
Rethinking this, I would prefer that we have a "friendlier" name for it :)

Basically, we're not trying to gatekeep or make it difficult for users to run the latest features, but we want a stable approach for backporting new features if agreed upon with the community.

Reply:
How about renaming it to "Experimental Development Workflow"?
Or even better, in today's meeting, we can vote on a suitable name for it, aiming for something less hostile sounding. 😄